docs/pages/api/access-plugin.mdx

```diff
@@ -0,0 +1,761 @@
+`proc.SpawnCriticalJob`, and wait for the job to terminate. We do this by
+receiving from a Go *channel*, which is a way for multiple concurrent routines
+to communicate with one another.
+
+### Initialize the API clients
+
+Now we have all the code we need to use the Teleport and Google Sheets API
+clients to listen for Access Request events and use them to maintain a
+spreadsheet. The final step is to start our program by initializing the API
+clients. 
+
+Add the following below the `run` function we defined in the last section:
+
+```go
+func main() {
+	ctx := context.Background()
+	svc, err := sheets.NewService(ctx, option.WithCredentialsFile("credentials.json"))
+	if err != nil {
+		panic(err)
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, initTimeout)
+	defer cancel()
+
+	creds := client.LoadIdentityFile("auth.pem")
+
+	teleport, err := client.New(ctx, client.Config{
+		Addrs:       []string{proxyAddr},
+		Credentials: []client.Credentials{creds},
+		DialOpts: []grpc.DialOption{
+			grpc.WithReturnConnectionError(),
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	gs := googleSheetsPlugin{
+		sheetsClient:   sheets.NewSpreadsheetsService(svc),
+		teleportClient: teleport,
+	}
+
+	if err := gs.run(); err != nil {
+		panic(err)
+	}
+}
+```
+
+The `main` function, the entrypoint to our program, initializes a
+`googleSheetsPlugin` and uses it run the plugin. 
+
+The function creates a Google Sheets API client by loading the credentials file
+you downloaded earlier at the relative path `credentials.json`.
+
+`client` is Teleport's library for setting up an API client. Our plugin does so
+by calling `client.LoadIdentityFile` to obtain a `client.Credentials`. It then
+uses the `client.Credentials` to call `client.New`, which connects to the
+Teleport Proxy Service specified in the `Addrs` field using the provided
+identity file.
+
+In this example, we are passing the `grpc.WithReturnConnectionError()` function
+call to `client.New`, which instructs the gRPC client to return more detailed
+connection errors.
+
+<Admonition type="warning">
+
+This program does not validate your credentials or Teleport cluster address.
+Make sure that: 
+
+- The identity file you exported earlier does not have an expired TTL
+- The value you supplied to the `Addrs` field in `teleport.Config` includes both
+  the host **and** the web port of your Teleport Proxy Service, e.g.,
+  `mytenant.teleport.sh:443`
+
+</Admonition>
+
+## Step 7/7. Test your plugin
+
+Now that you have written your plugin, run it to forward Access Requests from
+your Teleport cluster to Google Sheets. Execute the following command from
+within your project directory:
+
+```code
+$ go run main.go
+```
+
+Now that the plugin is running, create an Access Request:
+
+(!docs/pages/includes/plugins/create-request.mdx!)
+
+You should see the new Access Request in your spreadsheet with the `PENDING`
+state.
+
+Sign into the Teleport Web UI as your original user. In your spreadsheet,  click
```

Comment:
The first two sentences can be swapped, since you don't need to independently open the web UI and sign in before clicking the link; Teleport will prompt you to sign in and redirect from that link.